### DISCUSSION TITLE
工具栏按钮当内部出错后，按钮状态无法恢复使用。

### DISCUSSION THREAD

**Discussion by wenlino** [dotnetcore/BootstrapBlazor#3733] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

![7df8c3010a69ec44715e4be607748196](https://github.com/dotnetcore/BootstrapBlazor/assets/58806067/a8bb2d32-a9fc-44ee-be97-5e8ca4ecab1f)

**Comment by bb-auto** [comment]
@wenlino Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Reply by ArgoZhang** [reply]
@wenlino 我觉得这是一个设计组件的思路问题，我们组件内部基本上时不捕获消费者代码异常的，这样更方便开发发现问题定位问题，假如我们内部都写了 `try catch` 会导致看起来功能正常，开发者可能就忽略了问题。
